data-prepper-core/src/test/java/org/opensearch/dataprepper/event/DefaultBaseEventBuilderTests.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.event;
+
+import org.opensearch.dataprepper.model.event.DefaultEventMetadata;
+import org.opensearch.dataprepper.model.event.EventMetadata;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.CoreMatchers.not;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.util.Collections;
+import java.util.Map;
+import java.time.Instant;
+
+class DefaultBaseEventBuilderTests {
+    private DefaultBaseEventBuilder defaultBaseEventBuilder;
+    class TestDefaultBaseEventBuilder extends DefaultBaseEventBuilder {
```

Comment:
I think you can delete this.

Reply:
we have decided to make this  `DefaultBaseEventBuilder ` an abstract class so that it can't be instantiated, right? That's why I was extending the class here. Let me know if I misunderstood your comment here.

Reply:
Tried to mock DefaultBaseEventBuilder but that didn't make much sense because we are testing API of DefaultBaseEventBuilder.